docs/content/any/guides/data_filtering/index.md

```diff
@@ -0,0 +1,194 @@
+
+`Relation`s are a special type that tells oso how one Class is related to another. They specify what the related type is and how it's related.
+
+- `kind` is either "one" or "many". "one" means there is one related object and "many" means there is a list of related objects.
+- `other_type` is the class of the related objects.
+- `my_field` Is the field on this object that matches `other_field`.
+- `other_field` Is the field on the other object that matches `this_field`.
+
+The my_field / other_field relationship is similar to a foreign key. It lets oso know what fields to match up with building a query for the other type.
+
+# Example
+
+{{< literalInclude
+      dynPath="exampleBPath"
+      from="docs: begin-b1"
+      to="docs: end-b1"
+      fallback="no" >}}
+
+We now have two sets of query functions. Our `{{% exampleGet "buildQuery" %}}`
+function depends on the class but our `{{% exampleGet "execQuery" %}}` and
+`{{% exampleGet "combineQuery" %}}` functions are the same for all types so we
+can set them with `{{% exampleGet "setDataFilteringQueryDefaults" %}}`.
+
+{{< literalInclude
+      dynPath="exampleBPath"
+      from="docs: begin-b2"
+      to="docs: end-b2"
+      fallback="no" >}}
+
+{{< literalInclude
+      dynPath="policyBPath"
+      fallback="no" >}}
+
+{{< literalInclude
+      dynPath="exampleBPath"
+      from="docs: begin-b3"
+      to="docs: end-b3"
+      fallback="no" >}}
+
+# Evaluation
+When oso is evaluating data filtering methods it uses queries to fetch objects. If there are multiple types involved it will make multiple queries and substitute in the results when needed.
+In the above example we are fetching Repositories, but we are basing our fetch on some information about their related Organization. To resolve the query Oso first fetches the relevent Organizations (based in this case on role assignments), and then uses the `Relation` definition to substitute in their ids to the query for Repositories. This is the main reason to use `Relation`s, they let oso know how different classes are related so we can resolve data filtering queries.
+Relation fields also work when you are not using data filtering methods and are just using `authorize` or another method where you have an object to pass in. In that case the query functions are still called to get related objects so if you're using a `Relation` to a type, you must define query functions for that type.
+
+# Limitations
+
+There are a few limitations to what you can do while using data filtering. You can not call any methods on the passed in resource and you can not pass the resource as an argument to any methods. Many cases where you would want to do this are better handled by Relation fields.
+
+Some Polar expressions are not supported. `not`, `cut` and `forall` are not allowed in policies that want to use data filtering. Numberic comparisons with the `<` `>` `<=` and `>=` are not currently supported either.
```

Comment:
spelling

Reply:
numberic kinda sounds like it means the same thing